packages/api-contracts/generated/src/accessibilityInsightServiceScanAPI.ts

```diff
@@ -0,0 +1,280 @@
+/*
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
did you take a look at https://github.com/Azure/autorest/blob/master/docs/client/auth.md